backend/vcdat/app.py

```diff
@@ -76,7 +76,11 @@ def plot_template():
     tmpl = request.get_json()
     t = templ_from_json(tmpl)
     canvas = vcs.init(bg=True)
-    g = vcs.getboxfill()
+    g = vcs.createboxfill()
+    g.xmtics1 = {.5 * i: "" for i in range(1,20,2)}
+    g.xmtics2 = g.xmtics1
+    g.ymtics1 = g.xmtics1
+    g.ymtics2 = g.xmtics1
```

Comment:
@doutriaux1 I'm running into an issue where my attempt to plot the tics make it so no labels show up in the sample. I may need to steal some of your time to figure this out, I'll let you know after I dig into it some more.